packages/react-select/src/components/MultiValue.js

```diff
@@ -88,7 +88,9 @@ export type MultiValueRemoveProps = {
 export function MultiValueRemove({
   children,
   innerProps,
+  selectProps: { components = {} },
 }: MultiValueRemoveProps) {
+  const CrossIcon = useMemo(() => defaultComponents({ components }).CrossIcon, [components]);
```

Comment:
Instead of calling `defaultComponents` here, can we instead pass down the components like we do for `MultiValue` [here](https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/Select.js#L1544-L1548).